.github/workflows/ci.yaml

```
@@ -48,6 +48,7 @@ jobs:
   docs:
     needs: pre-commit
     runs-on: ubuntu-latest
+    if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)